aptos-move/framework/aptos-framework/sources/storage_gas.spec.move

```diff
@@ -77,4 +124,23 @@ spec aptos_framework::storage_gas {
     // ---------------------------------
 
     spec fun spec_calculate_gas(max_usage: u64, current_usage: u64, curve: GasCurve): u64;
+
+    spec schema AbortsIfNewGasCurve {
```

Comment:
For the convention, could you rename this to `NewGasCurveAbortsIf`?

Reply:
The modification is complete.

Reply:
Done.